test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
This patch does indeed build a single catch unit test executable, just as you expected.